src/DNSC/Iterative.hs

```diff
@@ -574,6 +576,47 @@ cacheSection rs rank = cacheRRSet
       insertRRSet <- asks insert_
       liftIO $ mapM_ ($ insertRRSet) rrss
 
+cacheEmptySection :: Domain -> Domain -> TYPE
```

Comment:
I would like to have a manual (comment) for this function.

Reply:
https://github.com/khibino/dns-resolver/pull/26/commits/ea37a076991bb5364f9980905863c0da8aa8991a